spec.html

```diff
@@ -26387,13 +26407,13 @@ <h1>Cyclic Module Records</h1>
             </tr>
             <tr>
               <td>
-                [[AsyncEvaluation]]
+                [[AsyncEvaluationOrder]]
               </td>
               <td>
-                a Boolean
+                an integer or ~unset~
               </td>
               <td>
-                Whether this module is either itself asynchronous or has an asynchronous dependency. Note: The order in which this field is set is used to order queued executions, see <emu-xref href="#sec-async-module-execution-fulfilled"></emu-xref>.
+                If this module is asynchronous or has an asynchronous dependency, this is a number representing the order in which modules that are asynchronous or have an asynchronous dependency start waiting on a pending promise.
```

Comment:
I don't find the "start waiting on a pending promise" description helpful, since the sorted list from GatherAvailableAncestors is most directly used to call out to ExecuteModule.